poi.config.js

```diff
@@ -21,7 +21,7 @@ const devServer = {
   },
   // Override using: `npm run dev:client -- --port <number>`
   port: 8081,
-  hotEntries: ['admin', 'student']
+  hotEntries: ['admin', 'learner']
```

Comment:
`main` is the name of the learner facing app so please rename the entry point to `main` as well